analyzers/tests/ITs.JsonParser.Test/IssueParserTest.cs

```diff
@@ -0,0 +1,186 @@
+﻿using SonarAnalyzer.Test;
+using SonarAnalyzer.Test.Helpers;
+
+namespace ITs.JsonParser.Test;
+
+[TestClass]
+public class IssueParserTest
+{
+    public TestContext TestContext { get; set; }
+
+    [TestMethod]
+    public void IssueParser_Execute_Single()
+    {
+        var root = TestDirectory();
+        var inputPath = Path.Combine(root, "read");
+        var outputPath = Path.Combine(root, "write");
+        Directory.CreateDirectory(Path.Combine(inputPath, "solution"));
+        Sarif.CreateReport(inputPath, "solution", "project", "net6.0", Sarif.CreateIssue("S100", "Message_1", "foo/bar/File1.cs", 1, 1));
+        var outFile = Path.Combine(outputPath, "solution", "S100-project-net6.0.json");
+
+        IssueParser.Execute(inputPath, outputPath);
+
+        VerifyResultFile(outFile, """
+            {
+              "Issues": [
+                {
+                  "Id": "S100",
+                  "Message": "Message_1.",
+                  "Location": {
+                    "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/foo/bar/File1.cs#L1",
+                    "Region": {
+                      "StartLine": 1,
+                      "StartColumn": 42,
+                      "EndLine": 1,
+                      "EndColumn": 69
+                    }
+                  }
+                }
+              ]
+            }
+            """);
+    }
+
+    [TestMethod]
+    public void IssueParser_Execute_Multiple()
+    {
+        var root = TestDirectory();
+        var inputPath = Path.Combine(root, "read");
+        var outputPath = Path.Combine(root, "write");
+        Directory.CreateDirectory(Path.Combine(inputPath, "solution1"));
+        Directory.CreateDirectory(Path.Combine(inputPath, "solution2"));
+
+        // solution1/project1-net6.0.json
+        Sarif.CreateReport(
+            inputPath,
+            "solution1",
+            "project1",
+            "net6.0",
+            Sarif.CreateIssue("S100", "Message_1", "foo/bar/File1.cs", 1, 1),
+            Sarif.CreateIssue("S100", "Message_2", "foo/bar/File1.cs", 42, 43),  // #L1-L2 location range
+            Sarif.CreateIssue("S142", "Message_3"));                             // null location
+
+        // solution1/project2-net6.0.json
+        Sarif.CreateReport(
+            inputPath,
+            "solution1",
+            "project2",
+            null, // No target framework specified
+            Sarif.CreateIssue("S200", "Message_1", "foo/bar/File1.cs", 0, 1)); // # Non-positive start line
+
+        // solution2/project-net6.0.json
+        Sarif.CreateReport(
+            inputPath,
+            "solution2",
+            "project",
+            "net6.0",
+            Sarif.CreateIssue("S300", "Message_1", "foo/bar/File1.cs", 1, 1));
```

Comment:
Make this S100 like the first one to prove that it will get it's own file based on the project name.
```suggestion
            Sarif.CreateIssue("S100", "Message_1", "foo/bar/File1.cs", 1, 1));
```